### PR TITLE
DS is actually OP now

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
## About The Pull Request

Creates a fancy blue energy sword. Added them to all DS packs. Buffed armor value and resistance of DS armor flags. This should let the admins do DS with more efficient work and not always with pulse gun but rather energy sword-shield tactics. Gives the DS to actually be DS. Also removes DS armor slowdown.

## Why It's Good For The Game

DS can actually be DS. Their gun wont suck and they will have varieties to choose from to fight. Their armor should provide better resistance. Got it approved by @BraveMole previously so putting this pr up. 

## Changelog
:cl:
add: Added blue energy sword
add: Added e-shields[expandable and retractable one] and e-swords to DS. More variety and content
balance: Buff DS armor values and resistance. Remove slowdown
admin: DS is actually OP now, use them with caution admins.
/:cl:

Wanted to add better looking DS sprites that kuro approved but the mob sprites didnt work, will try it later.

### CODE REVIEW

**Comment by ilikesaying** [comment]
still dies to fire + stunlock lol

**Comment by SpaceLoveSs13** [comment]
> still dies to fire + stunlock lol

The issue with fire is that it can disregard Armor. However I think ds provided more heat protection after a recent change and also not get put on flame easily. The stunlock should remain because melee stuns are somewhat legit, and if they manage to get a ds with melee stuns, they deserve  to win.

File: code/game/objects/items/weapons/energy.dm [hunk 1]
```diff
@@ -47,8 +47,8 @@
 	name = "energy sword"
 	desc = "May the force be within you."
 	icon_state = "sword0"
-	force = 3.0
-	throwforce = 5.0
+	force = 10.0
```
Comment (hyper2snyper) [inline code/game/objects/items/weapons/energy.dm:50]: remove the .0 while you are at it

File: tgmc.dme [hunk 2]
```diff
@@ -1511,11 +1511,11 @@
 #include "code\modules\mob\living\silicon\ai\logout.dm"
 #include "code\modules\mob\living\silicon\ai\multicam.dm"
 #include "code\modules\mob\living\silicon\ai\say.dm"
+#include "code\modules\mob\living\silicon\ai\vox_sounds.dm"
```
Comment (hyper2snyper) [inline tgmc.dme:1514]: doesnt do anything, but best practise to not change this here.

**Review by hyper2snyper — CHANGES_REQUESTED** [review]

File: tgmc.dme (hunk 2 above)
Comment (SpaceLoveSs13) [reply tgmc.dme:1514]: dont know how it even got here? Want me to remove it?

**Pull request merged**